Parsers/ASimFileEvent/test/Microsoft_Security_Event_ASimFileEvent_SchemaTest.csv

```diff
@@ -0,0 +1,106 @@
+﻿Result
```

Comment:
(0) Warning: 1 invalid value(s) (up to 10 listed) in 656144 records (100.0%) for field [EventProduct] of type [Enumerated]: ["Security Events"] (Schema:FileEvent)

Reply:
Event product is not there is documentation

Reply:
Please add Security Events in EventProduct field row for FileEvent schema in below file:
https://github.com/Azure/Azure-Sentinel/blob/master/ASIM/dev/ASimTester/ASimTester.csv